mhcflurry/class1_affinity_prediction/class1_affinity_predictor.py

```diff
@@ -119,6 +119,27 @@ def supported_peptide_lengths(self):
             max(lower for (lower, upper) in length_ranges),
             min(upper for (lower, upper) in length_ranges))
 
+    def merge(self, affinity_predictors=[]):
```

Comment:
Hm was thinking if you call merge and don't pass anything in dont want the entire thing to crash but on second thought is a bit unnecessary